CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.4]
+### Changed
+- Receive a callback to trigger `saveState`
```

Comment:
I'm having trouble understanding the purpose of this change, and the expectations for people using this package. What is the impact and how should usage be affected (e.g. is it breaking?)

Also, can this be applied to the main branch as a PR first? Or is this not applicable there?

Reply:
Yes, indeed it's a breaking change. I updated the changelog to include this info, better explain the change and link the related PRs.